source/info/bockowsk/Ebook.java

```diff
@@ -0,0 +1,54 @@
+package info.bockowsk;
+
+import java.io.*;
+
+class Ebook extends Book implements Comparable<Ebook> {
+
+    Ebook(String a, String t, String w) {
+        super(a,t,w);
+        String nazwa=a+"-"+t+".txt";
+        File plik1=new File(nazwa);
+        String linia;
+        try {
+            FileReader fileReader1=new FileReader(plik1);
```

Comment:
You should stick to one language in code. Don't mix polish and english names.

I strongly suggest english names - they allow any devleoper in the world to work with the source code.

Reply:
ack, I will stay with English names for variables, properties, methods etc.